content/salaries-and-benefits.md

```diff
@@ -44,6 +44,46 @@ As you progress in your teaching career, it's possible to move up through these
 | London fringe                 | £26,948 | £42,780 |
 | The rest of England and Wales | £25,714 | £41.604 |
 
+## Teacher benefits
+
+### Newly qualified teacher benefits
+ 
+A new 2-year package to support teachers at the start of their career, based on the [Early Career Framework](https://www.gov.uk/government/publications/supporting-early-career-teachers), launches nationally in September 2021.
+ 
+The support package includes:
+ 
+* funded 5% time off timetable in the second year of teaching, in addition to the existing 10% in the first year
+* a range of high-quality, freely available curricula and training materials underpinned by the Early Career Framework
+* funded training for mentors of early career teachers
+* funded time for mentors to support early career teachers 
+
+### Teaching and learning responsibility (TLR) payments
+
+If you take on extra ongoing responsibilities in your role, you could get a salary uplift.
```

Comment:
Worth hinting at what kind of responsibilities these might be?

Reply:
Done via guidance I've spotted, not sure if it is a bit clunky....